axlearn/common/flash_attention/gpu_attention.py

```diff
@@ -590,6 +590,9 @@ def _mha_backward(
 
     # NOTE: temporarily removed the "xla" branch, which seems unused.
     if backward_pass_impl == "triton":
+        # We must shrink the block size for float32 inputs to avoid OOM during bwd pass.
+        if jnp.float32 in (q.dtype, k.dtype, v.dtype):
+            block_q = block_k = 64
```

Comment:
Should we use min instead, in the case of block_q and block_k being less than than 64?
```
block_q = min(64, block_q)
block_k = min(64, block_k)
```

Not sure if there is any scenario where block_k and block_q are less than 64 though.

Reply:
looks like flash attention min is set to 64. So resolving this comment.
![image](https://github.com/user-attachments/assets/404c7a2c-8566-4388-bc3e-3b26946469d7)